packages/function-entry/src/input.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import debug from 'debug';
+import classNames from 'classnames';
+import range from 'lodash/range';
+import MuiInput from 'material-ui/Input';
+import TextField from 'material-ui/TextField';
+import { MuiThemeProvider, createMuiTheme, withStyles } from 'material-ui/styles';
+import { FormControl, FormHelperText } from 'material-ui/Form';
+import { indicators } from '@pie-lib/render-ui';
+import PropTypes from 'prop-types';
+
+const { Correct, Incorrect, NothingSubmitted } = indicators;
+
+const tags = {
+  'correct': Correct,
+  'incorrect': Incorrect,
+  'empty': NothingSubmitted
+}
+
+const log = debug('pie-elements:function-entry:input');
+
+class RawInput extends React.Component {
```

Comment:
Does this comp look similar to another comp we have? Is there an opportunity for re-use?

Reply:
There's definite opportunity for re-use here, as it's mostly the same component, I just wasn't exactly sure how to proceed since all of these are separate packages. Should we perhaps create a simple input component somewhere that we can re-use across various packages? I'd need to know where to create such a component.